transport/grpc/inbound.go

```diff
@@ -81,6 +66,55 @@ func (i *inbound) Start(service transport.ServiceDetail, d transport.Deps) error
 	return nil
 }
 
+/*
```

Comment:
Nit - This is the first time I've seen a block comment over a func... @prashantv is that idiomatic?